scripts/build/package.js

```diff
@@ -94,19 +94,26 @@ data.icons.forEach((icon) => {
   icons.push(icon);
 
   const iconObject = iconToObject(icon);
+  const iconExportName = slugToVariableName(icon.slug);
 
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
-  writeJs(jsFilepath, `module.exports=${iconObject};`);
+  const newImportMessage = `use "const {${iconExportName}} = require('simple-icons/icons')" instead`;
```

Comment:
Prettier adds spaces next to `{` `}` characters. Just a minor change but seems reasonable:

```suggestion
  const newImportMessage = `use "const { ${iconExportName} } = require('simple-icons/icons')" instead`;
```

Reply:
I originally had it like that to make the message smaller to reduce the impact on the package size, but it doesn't make much of a difference.

Reply:
It doesn't matter I guess, as this is temporal in v6